src/redux/actions/authActions.js

```diff
@@ -9,105 +9,122 @@ export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const GET_USER_INFO_FAILURE = 'GET_USER_INFO_FAILURE';
+export const GET_USER_INFO_SUCCESS = 'GET_USER_INFO_SUCCESS';
 
 function requestSignup() {
   return {
     type: SIGNUP_REQUEST,
-    receivedAt: Date.now()
+    lastUpdated: Date.now(),
+    isFetching: true
   };
 }
 
-function signupSuccess() {
+function signupFailure() {
   return {
-    type: SIGNUP_SUCCESS,
-    receivedAt: Date.now(),
-    isAuthenticated: true
+    type: SIGNUP_FAILURE,
+    lastUpdated: Date.now(),
+    isFetching: false
   };
 }
 
-function signupFailure() {
+function signupSuccess() {
   return {
-    type: SIGNUP_FAILURE,
-    receivedAt: Date.now()
+    type: SIGNUP_SUCCESS,
+    lastUpdated: Date.now(),
+    isAuthenticated: true,
+    isFetching: true,
   };
 }
 
 function requestLogin() {
   return {
     type: LOGIN_REQUEST,
-    receivedAt: Date.now()
+    lastUpdated: Date.now(),
+    isFetching: true
   };
 }
 
-function loginSuccess() {
+function loginFailure() {
   return {
-    type: LOGIN_SUCCESS,
-    receivedAt: Date.now(),
-    isAuthenticated: true
+    type: LOGIN_FAILURE,
+    lastUpdated: Date.now(),
+    isFetching: true
   };
 }
 
-function loginFailure() {
+function loginSuccess() {
   return {
-    type: LOGIN_FAILURE,
-    receivedAt: Date.now()
+    type: LOGIN_SUCCESS,
+    lastUpdated: Date.now(),
+    isAuthenticated: true,
+    isFetching: false,
   };
 }
 
 function requestLogout() {
   return {
     type: LOGOUT_REQUEST,
-    receivedAt: Date.now()
+    lastUpdated: Date.now()
   };
 }
 
-function logoutSuccess() {
+function logoutFailure() {
   return {
-    type: LOGOUT_SUCCESS,
-    user: null,
-    receivedAt: Date.now(),
-    isAuthenticated: false
+    type: LOGOUT_FAILURE,
+    lastUpdated: Date.now()
   };
 }
 
-function logoutFailure() {
+function logoutSuccess() {
   return {
-    type: LOGOUT_FAILURE,
-    receivedAt: Date.now()
+    type: LOGOUT_SUCCESS,
+    lastUpdated: Date.now(),
+    isAuthenticated: false,
+    user: null
   };
 }
 
-function requestUserInfo() {
+function getUserInfoRequest() {
   return {
-    type: REQUEST_USER_INFO,
-    receivedAt: Date.now()
+    type: GET_USER_INFO_REQUEST,
+    lastUpdated: Date.now(),
+    isFetching: true
   };
 }
 
-function receiveUserInfoSuccess(user) {
+function getUserInfoFailure() {
   return {
+    type: GET_USER_INFO_FAILURE,
+    lastUpdated: Date.now(),
     isFetching: false,
-    type: RECEIVE_USER_INFO_SUCCESS,
-    user: user,
-    receivedAt: Date.now()
   };
 }
 
-function receiveUserInfoFailure() {
+function getUserInfoSuccess(user) {
   return {
+    type: GET_USER_INFO_SUCCESS,
+    lastUpdated: Date.now(),
     isFetching: false,
-    type: RECEIVE_USER_INFO_FAILURE,
-    receivedAt: Date.now()
+    user: user,
   };
 }
 
-// TODO should fetching token be handled with a separate success/failure action?
-function fetchUserInfo() {
+/**************************** Logical functions ****************************/
```

Comment:
I like this commenting convention

Reply:
I call it the "Keliher" commenting convention.